src/main/java/org/rumbledb/expressions/module/MainModule.java

```diff
@@ -33,20 +33,20 @@
 public class MainModule extends Expression {
 
     private final Prolog prolog;
```

Comment:
For completeness, should we also refactor Prolog type with Expression type here?

Reply:
Actually a prolog is not an expression (but this part is not refactored yet).